### ISSUE TITLE
Allow search in "Project and External Dependencies"

### ISSUE THREAD

**Issue by skerit** [microsoft/vscode-java-pack#1276]
## Suggestion

I'm currently not able to search for something in the (decompiled) external dependencies.

## Use Cases

It's very useful for Minecraft modding.

**Comment by nightpool** [comment]
In my case, it's really important to be able to do *full* text search, just like I would on my normal code. The linked tickets seem to only be about adding support for searching for _method names_ to the existing cmd-T type search. While that would solve some of my issues, my dependencies often have e.g. string constants, JSON resources, or other full-text items I need to be able to find—not just class names or method names.

It looks like this would maybe require an extension to the language server specification? I'm not sure how e.g. Eclipse and IntelliJ handle this, but they both have this feature.

**Comment by chagong** [comment]
@nightpool, yeah, pls refer to https://github.com/redhat-developer/vscode-java/issues/1283, currently you may use type "#" on the top search box of vscode (Ctrl+T / Cmd+T) to search symbols.
